10_onion_routing.asciidoc

```diff
@@ -331,6 +331,26 @@ Thus, they will both get the same result which they can use as a shared key to s
 
 ****
 
+A unique trait of Sphinx as a mix-net packet format is that rather than include a distinct session key for each hop in the route, which would increase the size of the mix-net packet dramatically, instead a clever _blinding_ scheme is used deterministically randomize the session key at each hop.
```

Comment:
I think this sentence would read clearer this way (crossed out words represent removal and bolded words represent additions):
"A unique trait of Sphinx as a mix-net packet format is that rather than include a distinct session key for each hop in the route, which would increase the size of the mix-net packet dramatically, i̶n̶s̶t̶e̶a̶d̶ a clever _blinding_ scheme is used **to** deterministically randomize the session key at each hop."

Reply:
Thanks, accepted the suggestion